torchquad/integration/vegas.py

```diff
@@ -0,0 +1,174 @@
+from .base_integrator import BaseIntegrator
+from .utils import setup_integration_domain
+
+from .vegas_map import VEGASMap
+from .vegas_stratification import VEGASStratification
+
+import torch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class VEGAS(BaseIntegrator):
+    """Vegas Enhanced in torch. Refer to https://arxiv.org/abs/2009.05112.
```

Comment:
Sorry for not noticing this the first time, but shouldn't it be `VEGAS Enhanced` here as well?